src/docs/patterns/appbar.mdx

```diff
@@ -1,43 +1,86 @@
+In some situations, you may want to present more than just a title and subtitle in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a taller app bar that will shrink as the user scrolls down the page until it becomes a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used for web.
 
-## Search Bar
+The collapsible appbar should transition smoothly between the expanded and collapsed states — this means you will need to account for how the app bar will look throughout the transition. Because elements will disappear as the app bar shrinks, you'll need to use dynamic text scaling, opacity transitions, and / or other techniques to do this smoothly, depending on your content.
+
+<Divider />
+
+# Search Bar
 
 <ImageGrid
     images={[Image3, Image4]}
     caption={'When the search icon is clicked, a white search bar will slide in and cover the default tool bar. '}
 />
 <DemoCard repository={'search-bar'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fsearch-bar%2Fsearch-bar.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
-A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white search bar slides in from the right to cover the default app bar. As the user enters text into the search field, the data is filtered in real-time. A back icon will close the Search Bar and clear the search results.
+
+## Auto-complete
+
+For more advanced searching, you may wish to include an auto-complete feature. You can implement autocomplete by including a contextual menu below the search bar with suggested search items. Alternatively, you can use type-ahead to automatically update the recommended search query in the search field as the user types (Google search does this).
 
 <Divider />
```

Comment:
One thing I was wondering — why are ng material and MUI both using 64px as the app bar height? I don't think the 64px is mentioned anywhere in Material's design spec.

Reply:
Good question...i don't know. It's possible it was part of their specs before and got removed and the component libraries just carried on.

Reply:
Hmmm, so should we shy away from the desktop dimension then?

Reply:
And can we have an img for collapsible app bar?

Reply:
I wouldn't shy away from it, unless we're planning on overhauling our themes to override the default behaviors. I don't see a reason to at this point in time. I think the 64px works better when you have a subtitle (56 feels cramped)...material doesn't do subtitles in toolbars as far as I can tell.

Reply:
I agree on the 64px works better with the subtitle part. Just be aware that DSM and all other design files are using 56px